signer/tests/integration/transaction_coordinator.rs

```diff
@@ -6161,26 +6163,37 @@ where
     regtest::p2tr_sign_transaction(&mut deposit_tx, 0, &[utxo], &depositor.keypair);
 
     let mut requests = vec![];
+    let mut infos = vec![];
     for (index, amount) in amounts.iter().enumerate() {
         let req = CreateDepositRequest {
             outpoint: bitcoin::OutPoint::new(deposit_tx.compute_txid(), index as u32),
             deposit_script: deposit_script.clone(),
             reclaim_script: reclaim_script.clone(),
         };
 
+        infos.push(DepositInfo {
+            outpoint: req.outpoint,
+            max_fee,
+            deposit_script: req.deposit_script,
+            amount: *amount,
+            reclaim_script: req.reclaim_script,
+            signers_public_key,
+            recipient: deposit_inputs.recipient.clone(),
+            lock_time: bitcoin::relative::LockTime::ZERO,
```

Comment:
nit: shouldn't the locktime be 50?

Reply:
Hmmm, is it ok that tests didn't catch it 🤔